### PR TITLE
Reworks deploy condition so it triggers properly

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ jobs:
       script:
         - make cfn.lint
     - stage: deploy
-      if: branch = master AND type = push AND repo = env(DEPLOY_SLUG)
+      if: branch = env(RELEASE_BRANCH) AND tag IS blank
```
Comment (eemperor) [inline .travis.yml]: Will this condition ever pass?  Will `tag` ever be blank?

My interpretation is that 'tag' would always contain the prior version's tag?

Comment (lorengordon) [reply .travis.yml]: `tag` is specific to the event/commit in the build...

* https://docs.travis-ci.com/user/conditions-v1#integration

> Predicates:
>
> `tag IS present`
> `tag IS blank`

* https://docs.travis-ci.com/user/environment-variables/#default-environment-variables

> TRAVIS_TAG: If the current build is for a git tag, this variable is set to the tag’s name.

Comment (eemperor) [reply .travis.yml]: Thanks for the links.  Tags aren't present until its applied after the deploy.

Comment (lorengordon) [reply .travis.yml]: Right, this deploy stage is intended to create the tag, when the version has been incremented and after the change is merged to master. It was working the way it was before, but travis changed how they handle conditions and it stopped working... 😁

**Pull request merged**